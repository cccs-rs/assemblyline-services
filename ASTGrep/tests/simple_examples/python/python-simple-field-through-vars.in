vars(zlib)['\x64\x65\x63\x6f\x6d\x70\x72\x65\x73\x73']
vars(obj1)[b'some_method']
