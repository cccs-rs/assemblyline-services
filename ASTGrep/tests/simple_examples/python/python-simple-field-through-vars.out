zlib.decompress
obj1.some_method
